projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.ts

```diff
@@ -176,10 +176,8 @@ export class QuoteReviewComponent implements OnInit, OnDestroy {
   }
 
   getFormContent(cart: any): any {
-    if (cart?.deliveryOrderGroups[0]?.entries[0]?.formData?.length > 0) {
-      return JSON.parse(
-        cart.deliveryOrderGroups[0].entries[0].formData[0].content
-      );
+    if (cart?.entries[0]?.formData?.length > 0) {
+      return JSON.parse(cart.entries[0].formData[0].content);
```

Comment:
Just a  friendly reminder that deliveryOrderGroups are not need when accessing entries on cart :)